Makefile

```diff
@@ -6,7 +6,7 @@ export SHELL = /bin/bash
 
 # Settings
 MAKEFILES=Makefile $(wildcard *.mk)
-JEKYLL=bundle install --path .vendor/bundle && bundle update && bundle exec jekyll
+JEKYLL=bundle config set path '.vendor/bundle' && bundle install && bundle update && bundle exec jekyll
```

Comment:
```
bundle config set path '.vendor/bundle'
```
sets this configuration globally (that is, in `~/.bundle/config` file). Let's add `--local` flag so that it reads:

```
bundle config --local set path .vendor/bundle
```

Reply:
Also, we might not need single quotes around `.vendor/bundle`